### PR TITLE
suggest way to not build when packing using a nuspec

### PR DESCRIPTION
**Pull request by rohit21agrawal**

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/rohit21agrawal/docs.microsoft.com-nuget/blob/patch-4/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803131937382203-755%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5fxEwAr9s4aHBjUOEQ8CFUVUTHPNNONVsUp%2F8DDIn5Q%3D&st=2018-03-13T19%3A33%3A31Z&se=2018-04-13T19%3A38%3A31Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/rohit21agrawal/docs.microsoft.com-nuget/blob/patch-4/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803131941320335-755%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=eFbY4XGcziPyUnUA4NzEjyHUlEoPud35ql7wTzA9x48%3D&st=2018-03-13T19%3A37%3A26Z&se=2018-04-13T19%3A42%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/msbuild-targets.md [hunk 1]
```diff
@@ -208,6 +208,8 @@ If using MSBuild to pack your project, use a command like the following:
 msbuild /t:pack <path to .csproj file> /p:NuspecFile=<path to nuspec file> /p:NuspecProperties=<> /p:NuspecBasePath=<Base path> 
 ```
 
+Please note that packing a nuspec using dotnet.exe or msbuild also leads to building the project by default. This can be avoided by passing ```--no-build``` property to dotnet.exe, which is the equivalent of setting ```<NoBuild>true</NoBuild> ``` in your project file.
```
Comment (jaredpar) [inline docs/reference/msbuild-targets.md]: Have you tried this locally? It doesn't work. 

C:\Program Files\dotnet\sdk\2.2.0-preview1-007622\Sdks\Microsoft.NET.Sdk\build\Microsoft.PackageDependencyResolution.targets(327,5): error : Assets file 'C:\Users\jaredpar\temp\obj\project.assets.json' not found. Run a NuGet package restore to generate this file. [C:\Users\jaredpar\temp\example.csproj]
C:\Program Files\dotnet\sdk\2.2.0-preview1-007622\Sdks\Microsoft.NET.Sdk\build\Microsoft.PackageDependencyResolution.targets(167,5): error : Assets file 'C:\Users\jaredpar\temp\obj\project.assets.json' not found. Run a NuGet package restore to generate this file. [C:\Users\jaredpar\temp\example.csproj]

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/rohit21agrawal/docs.microsoft.com-nuget/blob/patch-4/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803132136375863-755%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=DFQxkMMoyV%2Bru9ggwxzsZPmpePuL3XtE8%2Bbv3CMOnNc%3D&st=2018-03-13T21%3A32%3A29Z&se=2018-04-13T21%3A37%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/msbuild-targets.md (hunk 1 above)
Comment (rohit21agrawal) [reply docs/reference/msbuild-targets.md]: updated, and provided an example too.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/rohit21agrawal/docs.microsoft.com-nuget/blob/patch-4/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803132138010676-755%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=tSf6QBM3Knm26jNFjEy1Vu8EgajfL7DinVZc6MprTXU%3D&st=2018-03-13T21%3A33%3A52Z&se=2018-04-13T21%3A38%3A52Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/reference/msbuild-targets.md [hunk 2]
```diff
@@ -208,6 +208,23 @@ If using MSBuild to pack your project, use a command like the following:
...
 
+Please note that packing a nuspec using dotnet.exe or msbuild also leads to building the project by default. This can be avoided by passing ```--no-build``` property to dotnet.exe, which is the equivalent of setting ```<NoBuild>true</NoBuild> ``` in your project file, along with setting ```<IncludeBuildOutput>false</IncludeBuildOutput> ``` in the project file
+
+An example of a csproj file to pack a nuspec file is:
+
+```
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <NoBuild>true</>
```
Comment (jaredpar) [inline docs/reference/msbuild-targets.md]: Closing tag is incorrect. Needs to be `</NoBuild>`

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/reference/msbuild-targets.md](https://github.com/rohit21agrawal/docs.microsoft.com-nuget/blob/patch-4/docs/reference/msbuild-targets.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C3%5C14%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201803141800271000-755%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NbnZVxSeU4VUnS3WyoqjgObeonoZpqp%2BKWTo%2FPiT2HQ%3D&st=2018-03-14T17%3A56%3A17Z&se=2018-04-14T18%3A01%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by rohit21agrawal** [comment]
@kraigb can you help merge this?

**Pull request merged**